apps/ledger-live-desktop/src/renderer/init.jsx

```diff
@@ -159,6 +157,11 @@ async function init() {
 
     events({ store });
 
+    // trigger the basic "ping" command to ensure internal process correctly works
+    await command("ping")()
+      .pipe(timeout(10000))
+      .toPromise();
```

Comment:
Is 10s a bit too short for a timeout that would crash LLD in prod ? I feel this could happen to some of our users

Reply:
i initially assume it would be instant but you are right (see my main comment on PR) we can move to 1min, then i need to see how we can move this code somewhere else so it doesn't block the UI